modules/indexing/src/test/java/org/apache/ignite/internal/metric/SqlStatisticOffloadingTest.java

```diff
@@ -80,7 +80,7 @@ public void testOffloadStats() {
         // Ensure that metrics increased.
         Metrics m1 = withdrawMetrics();
 
-        Metrics expected = m0.add(Metrics.of(6249, 6249, 1, 16878, 16878, 1)); // Expected numbers of written bytes.
+        Metrics expected = m0.add(Metrics.of(6249, 6249, 1, 16878, 16878, 2)); // Expected numbers of written bytes.
```

Comment:
why was test expectation changed ?

Reply:
same question about `JdbcQueryQuotaTest`, btw, why was memory quota doubled there ?

Reply:
because now all local map queries and reduce shares the common tracker, so they shares common quota